backend/src/modules/doubles-tournament/core/domain/QueuedMatch.ts

```diff
@@ -0,0 +1,70 @@
+import { TeamId } from './TeamId';
+import { TournamentId } from './TournamentId';
+import { DoublesTournament } from './DoublesTournament';
+import { DomainCommandResult } from '../../../../shared/core/domain/DomainCommandResult';
+import { MatchesQueue } from './MatchesQueue';
+import { MatchWasQueued } from './event/MatchWasQueued';
+
+export class QueuedMatch {
+  readonly matchNumber: number;
```

Comment:
Fajnie z MatchNumbera tez zrobic value object, bo np. wiemy, ze to nie jest kazdy numer, ze np. minimum to jest 1.